roles/update-access/test/inventory

```diff
@@ -0,0 +1,15 @@
+# Staging inventory file
+[all:vars]
+
+# create encrypted password with:
+# python -c 'import crypt; print crypt.crypt("password", crypt.mksalt(crypt.METHOD_SHA512))'
+
+# Text: 'password'=> encrypted: '$6$z8S0Iw6iAVsJRY19$/YKREfgO2yxwbuayGmJ.w8MQb0gtVwjBaJr6yQp8alLuFj1S/YoKFWORsq/MyIjLhoBkj29TxUj.EdOT4PLD1.'
+user_name_password:$6$z8S0Iw6iAVsJRY19$/YKREfgO2yxwbuayGmJ.w8MQb0gtVwjBaJr6yQp8alLuFj1S/YoKFWORsq/MyIjLhoBkj29TxUj.EdOT4PLD1.
```

Comment:
this is invalid yaml - need a space after the colon as otherwise Ansible won't successfully run and it will error out with a parsing problem.